### PR TITLE
Feature/exhibition index page

### PR DESCRIPTION
**Pull request by CBU1980**
Added a

- basic exhibition teaser
- featured exhibitions list
- upcoming exhibitions list

ping @martzoukos

### CODE REVIEW

File: source/partials/components/exhibition-teaser.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<a href="/exhibitions/<%= exhibition[:slug] %>" class="exhibition-teaser <% if locals.has_key?(:featured) %>exhibition-teaser--featured<% end %>">
```
Comment (martzoukos) [inline source/partials/components/exhibition-teaser.html.erb]: Use `<%= link_to` helpers.

File: source/partials/components/exhibition-header.html.erb [hunk 2]
```diff
@@ -11,8 +11,14 @@
...
       <span><%= t('exhibitions.header.label') %></span>
       <% if exhibition.start_date.present? %>
-        <span><%= exhibition.start_date.strftime('%d.%m.%Y, %H:%M') %></span>
-        <span><%= exhibition.end_date.strftime('%d.%m.%Y, %H:%M') %></span>
+        <span>
+          <%= t('exhibitions.teaser.from') %>
+          <%= exhibition.start_date.strftime('%d.%m.%Y') %>
+        </span>
+        <span>
+          <%= t('exhibitions.teaser.from') %>
```
Comment (martzoukos) [inline source/partials/components/exhibition-header.html.erb]:
```
<%= t('exhibitions.teaser.from', data: exhibition.end_date.strftime('%d.%m.%Y') ) %>
```

and in en.yml:

```
exhibitions:
    teaser:
        from: 'text text text %{title}'
```

File: source/partials/components/exhibition-teaser.html.erb (hunk 1 above)
Comment (CBU1980) [inline source/partials/components/exhibition-teaser.html.erb]: Done.

File: source/partials/components/exhibition-header.html.erb (hunk 2 above)
Comment (CBU1980) [inline source/partials/components/exhibition-header.html.erb]: Nice. Done as well.

**Pull request merged**